src/main/java/picard/sam/BamErrorMetric/ErrorMetric.java

```diff
@@ -0,0 +1,52 @@
+package picard.sam.BamErrorMetric;
+
+import htsjdk.samtools.util.QualityUtil;
+import picard.analysis.MergeableMetricBase;
+
+/**
+ * Created by farjoun on 6/26/18.
+ */
+public class ErrorMetric extends MergeableMetricBase {
+
+    public static void setPriorError(double priorError) {
```

Comment:
Not sure what the picard convention is, but it's typical in Java code to sort fields before methods